documentation/book/security.adoc

```diff
@@ -1,61 +1,75 @@
-More precisely, each broker has two different certificates (and related private keys), one for interbroker communication  and one for communication with clients.
-These broker certificates are signed using different CAs: The "internal-ca" signs the certificates used for interbroker communication, and the "clients-ca" signs the certificates used for client communication.
+All component certificates are signed by a Certification Authority (CA) called _cluster CA_.
+Another CA is used for authentication of Kafka clients connecting to Kafka brokers.
+This CA is called _clients CA_.
 The CAs themselves use self-signed certificates.
-The "internal-ca" is used across multiple clusters, that is, there is a single "internal-ca" for each instance of a cluster operator. 
-The "clients-ca" is specific to a particular Kafka cluster.
 
 All the generated certificates are saved as Secrets in the {ProductPlatformName} cluster, named as follows:
 
-* `internal-ca`: contains the private and public keys, so the self-signed certificate, used for signing broker certificates used for interbroker communication. It is common to all the Kafka clusters deployed by the Cluster Operator.
-* `<cluster-name>-kafka-clients-ca`: contains the private and public keys, so the self-signed certificate, used for signing broker certificates used for communicating with clients. It is specific for each deployed Kafka cluster as specified in the <cluster-name> prefix.
-* `<cluster-name>-kafka-cert`: contains only the public keys, so the self-signed certificate, used for signing broker certificates used for communicating with clients. This Secret should be used by the final Kafka cluster user in order to extract this certificate to put into the the client's truststores for verifying broker identities (server authentication).
-* `<cluster-name>-kafka-brokers-internal`: contains all the brokers private and public keys (certificates signed with "internal-ca") used for interbroker communication.
-* `<cluster-name>-kafka-brokers-clients`: contains all the brokers private and public keys (certificates signed with specific cluster "clients-ca") used for communicating with clients.
+`<cluster-name>-cluster-ca`::
+Contains the private and public keys of the cluster CA wich is used for signing server certificates for the Kafka and Strimzi components (Kafka brokers, Zookeeper nodes and so on).
+`<cluster-name>-cluster-ca-cert`::
+Contains only the public key of the cluster CA.
+The public key should be used by Kafka clients to verify the identity of the Kafka brokers they are connecting to.
+`<cluster-name>-clients-ca`::
+Contains the private and public keys of the clients CA.
+The clients CA is used for TLS client authentication of Kafka clients when connecting to Kafka brokers.
+`<cluster-name>-clients-ca-cert`::
+Contains only the public key of the client CA.
+The public key is used by the Kafka brokers to verify then identity of Kafka clients when TLS client authentication is used.
```

Comment:
Looks like a typo - change "to verify then identity" to  "to verify the identity"